### DISCUSSION TITLE
功能建议

### DISCUSSION THREAD

**Discussion by rebron1900** [usememos/memos#29] [Ideas]
1. 希望有图床功能，小vps存图片有点受不了。
2. 输入#号后希望可以有标签列表。
3. 还是希望能有微信渠道的发送方式
4. 生成的分享图片可以用筛选后的结果生成。

**Comment by yylqing** [comment]
您好，我已收到您发的邮件，将会尽快查看

**Comment by rebron1900** [comment]
我们那个博客群四五个都开始用了，还在安利中。

**Comment by boojack** [comment]
> 1. 希望有图床功能，小vps存图片有点受不了。
> 2. 输入#号后希望可以有标签列表。
> 3. 还是希望能有微信渠道的发送方式
> 4. 生成的分享图片可以用筛选后的结果生成。

1. 可以通过直接粘贴图片地址（也就是图床地址）到输入框，会自动解析成图片，并隐藏地址字符串；
2. WIP;
3. 不会直接再加入到主程序中，但可以弄成 plugin 模式，用户自行配置（工作量偏大）
4. 指的是：给当前所有符合筛选条件的 Memo 生成图片的功能吗

**Comment by rebron1900** [comment]
1. 不知道可以配置对应的图床呢？我用的[twikoo](https://twikoo.js.org)里面有个选项就是适配了多个图床。
3. 谢谢，希望还是能有这个功能，不管是以什么形式体现。
4. 是的，根据当前的筛选生成图片。
